src/main/resources/db/migration/V2__varselstatus_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE varselstatus (
+    id                      INT             GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```

Comment:
Er det noen grunn til at dette er en int fremfor UUID? Jeg trodde kanskje konvensjonen i NAV var UUID for ID-er? :)

Reply:
Svarte på denne på slack forresten :) Kort fortalt er det bare et løpenummer, det brukes ikke til noenting. Da føles det litt overkill å bruke en uuid til det :)